code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -264,20 +264,19 @@
 
 	var/mob/living/carbon/xenomorph/ravager/rager = owner
 
-	var/rage_health_threshold = rager.maxHealth * 0.5 //Need to be at 50% of max hp or lower to rage
-	if(rager.health > rage_health_threshold)
+	if(rager.health > rager.maxHealth * 0.5) //Need to be at 50% of max hp or lower to rage
 		if(!silent)
-			to_chat(rager, "<span class='xenodanger'>Our health isn't low enough to rage! We must take [rager.health - rage_health_threshold] more damage!</span>")
+			to_chat(rager, "<span class='xenodanger'>Our health isn't low enough to rage! We must take [rager.health - (rager.maxHealth * 0.5)] more damage!</span>")
 		return FALSE
 
 
 /datum/action/xeno_action/rage/action_activate()
 	var/mob/living/carbon/xenomorph/ravager/X = owner
 
-	rage_power = (1-(X.health/X.maxHealth)) * 0.5 //Calculate the power of our rage; scales with difference between current and max HP
+	rage_power = (1-(X.health/X.maxHealth)) * RAVAGER_RAGE_POWER_MULTIPLIER //Calculate the power of our rage; scales with difference between current and max HP
```

Comment:
correct me if i'm wrong, but this 0.5 is the same as the 50% health?

So the define should be shared